app/models/crop.rb

```diff
@@ -354,6 +335,23 @@ def self.search(query)
     end
   end
 
+  private
+
+  def add_names_to_list(names_to_add, list_name, col_name)
+    cropbot = Member.find_by_login_name('cropbot')
+    names_to_add.each do |n|
+      if self.send("#{list_name}_names").exists?(["#{col_name} LIKE ?", n])
```

Comment:
If alternate_names and scientific_names both just called their column `name` this'd be SO much easier. 

The reason I'm wondering about a better way is that Code Climate's warning about SQL injection. Now, I don't think injection is *actually* an issue here, since this is in a private function in the model operating on a local csv file, but...figured I'd check.